pdctl/command/operator.go

```diff
@@ -198,6 +199,35 @@ func addPeerCommandFunc(cmd *cobra.Command, args []string) {
 	postJSON(cmd, operatorsPrefix, input)
 }
 
+// NewMergeRegionCommand returns a command to transfer leader.
+func NewMergeRegionCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "merge-region <source_region_id> <target_region_id>",
```

Comment:
how can we get source and sibling target ID easily?

Reply:
okay, I will add a command for querying sibling region id